modules/compliance-new-tailored-profiles.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-tailor.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-new-tailored-profiles_{context}"]
+= Creating a new tailored profile
+
+You can write a tailored profile from scratch using the TailoredProfile object. Set an appropriate `title` and `description` and leave the `extends` field empty. Indicate to the Compliance Operator what type of scan will this custom profile generate:
+
+* Node scan: Scans the Operating System.
+* Platform scan: Scans the OpenShift configuration.
+
+Set the following annotation on the TailoredProfile object:
+
+.Example `new-profile.yaml`
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: TailoredProfile
+metadata:
+  name: new-profile
+spec:
+  extends: <1>
+  description: My custom profile <2>
+  title: Custom profile <3>
+compliance.openshift.io/product-type: Node/Platform <4>
+----
+<1> Because this is a new TailoredProfile object, you must leave the `extends` field empty.
```

Comment:
It is wrong. It is an optional parameter. User could decide to enable the rules in enableRules list, or extended to the whole profile the enabled rules belongs to.